pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -72,7 +76,7 @@ type stateData struct {
 
 	// The indices of all claims that:
 	// - are allocated
-	// - use delayed allocation
+	// - use delayed allocation or a builtin controller
```

Comment:
```suggestion
	// - use delayed allocation or the builtin controller
```
?

Is there more than one?

Reply:
Right now, there's only one (the one for "named resources"). Others might be added in the future. Should I mentioned that here or keep it vague?

Reply:
Yes, that's a different way to think about this. So let's replace with "the builtin controller".

Reply:
I see, I was thinking of kubernetes controllers, but it's actually referring a tiny "controller" inside scheduler.

Maybe I would name them models instead of controllers. But that's a nit.